det3d/datasets/argoverse/argoverse_common.py

```diff
@@ -0,0 +1,330 @@
+import copy
+import json
+import os
+from typing import Dict, Tuple, List
+
+import numpy as np
+import torch
+from argoverse.data_loading.object_label_record import json_label_dict_to_obj_record
+from argoverse.data_loading.pose_loader import get_city_SE3_egovehicle_at_sensor_t
+from argoverse.utils import ply_loader, se3
+from scipy import interpolate
+from scipy.spatial.transform import Rotation, RotationSpline
+
+YSize = 80
+XSize = 144
+
+
+class Label:
+    """
+    Class to store the bounding boxes and label class of an occupancy grid
+    """
+
+    def __init__(self, record, label):
+        self.record = record
+        self.boxes = None
+        self.options = {"BACKGROUND": 0, "VEHICLE": 1}
+        self.label_class = label
+        self.label_num = self.options[self.label_class]
+
+
+def load_all_clouds(dataset: str, log_id: str, timestamp_list: List[int]) -> Dict:
+    """Loads all point clouds from the Argoverse Tracking set
```

Comment:
I think this docstring may be out-of-date, perhaps it should be `Loads all point clouds from a specified log in the Argoverse Tracking dataset`